src/cmd/cmd.cpp

```diff
@@ -442,6 +442,11 @@ int main(int argc, char **argv)
     loop.exec();
 #endif
 
+    if (job->reply()->error() != QNetworkReply::NoError){
+        std::cout<<"Authentication Error\n";
```

Comment:
Should I change it to "Problem connecting to server"? Even if there is any other error , the next steps (database creation etc) shouldn't happen.